Docs/ProjectSpec.md

```diff
@@ -130,6 +132,81 @@ options:
     tvOS: "10.0"
 ```
 
+### Breakpoints
+
+- [x] **type**: **String** - Breakpoint type
+    - `FileBreakpoint`: file brakpoint
+    - `ExceptionBreakpoint`: exception breakpoint
+    - `SwiftErrorBreakpoint`: swift error breakpoint
+    - `OpenGLErrorBreakpoint`: OpenGL breakpoint
+    - `SymbolicBreakpoint`: symbolic breakpoint
+    - `IDEConstraintErrorBreakpoint`: IDE constraint breakpoint
+    - `IDETestFailureBreakpoint`: IDE test failure breakpoint
+- [ ] **enabled**: **Bool** - indicates whether it should be active
+- [ ] **ignoreCount**: **String** - indicates how many times it should be ignored before stopping
+- [ ] **continueAfterRunningActions**: **Bool** - indicates if should automatically continue after evaluating actions
+- [ ] **timestamp**: **String** - breakpoint timestamp
+- [ ] **filePath**: **String** - breakpoint file path (only used by file breakpoints)
+- [ ] **startingColumn**: **String** - breakpoint starting column (only used by file breakpoints)
+- [ ] **endingColumn**: **String** - breakpoint ending column (only used by file breakpoints)
+- [ ] **startingLine**: **String** - breakpoint starting line (only used by file breakpoints)
+- [ ] **endingLine**: **String** - breakpoint ending line (only used by file breakpoints)
+- [ ] **breakpointStackSelectionBehavior**: **String** - breakpoint stack selection behavior
+- [ ] **symbol**: **String** - breakpoint symbol (only used by symbolic breakpoints)
+- [ ] **module**: **String** - breakpoint module (only used by symbolic breakpoints)
+- [ ] **scope**: **String** - breakpoint scope (only used by exception breakpoints)
+- [ ] **stopOnStyle**: **String** - indicates if should stop on style (only used by exception breakpoints)
+- [ ] **condition**: **String** - breakpoint condition
+- [ ] **locations**: **[Location]** - breakpoint locations (currently not used by Apple)
+- [ ] **actions**: **[[Breakpoint Action](#breakpoint-action)]** - breakpoint actions
+
+```yaml
+breakpoints:
+  - type: ExceptionBreakpoint
+    enabled: true
+    ignoreCount: 0
```

Comment:
You can see this `ignoreCount` has been set to an int which makes sense, hence the above comment